Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu

```

Comment:
I tried to use the `jekyll/minimal` and `jekyll/jekyll` images but neither seemed to play ball out of the box, maybe something to do with us using GitHub pages specifically.